src/styles/book.css

```diff
@@ -0,0 +1,64 @@
+.bookBox {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.bookTitle {
+  width: 90%;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.rankData {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 50%;
+}
+
+.rankNumber {
+  font-size: 5vh;
+}
+
+.arrowRankBox {
+  width: auto;
+  height: 100%;
+}
+
+.arrowRankBox img {
+  height: 4vh;
+  object-fit: contain;
+  padding: 0;
+}
+
+.bookCoverBox {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.bookCoverBox img {
+  padding: 2vh;
+  height: 27vh;
+  object-fit: contain;
+}
+
+.titleAuthorBook {
+  width: 50%;
+  font-size: 2vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.weeksList {
+  padding: 2vh;
+  font-weight: 300;
+  font-size: 2vh;
+}
+
+.titleAuthorBook h2 {
+  margin-bottom: 0;
+}
```

Comment:
As part of the project requirements, kindly make sure that your design follows the design guidelines in terms of the layout (composition and space between elements). There are some slight differences I see between your design and that of the actual design as shown below: 

- [ ] Your design is missing the  small white arrow in a circle to the right side of the details card. 
- [ ] The background of the individual details card is light  whilst the next one is darker and it follows that order downwards as shown below :point_down: 

Kindly check and update your design accordingly. 

### Your design 

![Screenshot from 2023-03-16 23-53-03](https://user-images.githubusercontent.com/65233860/225781249-7615ec13-0bd9-45c1-b062-41b5493b0047.png)


### Actual design 


![Screenshot from 2023-03-16 23-55-26](https://user-images.githubusercontent.com/65233860/225781258-8f6eaba6-00b0-4892-ba25-cabeccb31a34.png)